ui/src/views/Tasks/CreateTask/index.jsx

```diff
@@ -265,6 +286,33 @@ export default class CreateTask extends Component {
           ) : (
             <Fragment>
               <ErrorPanel error={createdTaskError} />
+              {this.state.recentTaskDefinitions &&
```

Comment:
Move the list so that it ends up being below the editor.

Reply:
Now that `recentTaskDefinitions` is an array by default. You can simply do:

```jsx
{Boolean(recentTaskDefinitions) && (
 // ...
)}
```

In the beginning of the render method, you'll also want to add `recentTaskDefinitions` so that you don't have to do `this.state.recentTaskDefinitions`.

Reply:
Also, add a `<br />` element between the editor and the list.